### PR TITLE
Fixes for breakpoints

### PR DESCRIPTION
**Pull request by vforge**
WORK IN PROGRESS

DO NOT MERGE

Fixes for:
https://wikia-inc.atlassian.net/browse/CONCF-534
https://wikia-inc.atlassian.net/browse/CONCF-555
https://wikia-inc.atlassian.net/browse/CONCF-562
https://wikia-inc.atlassian.net/browse/CONCF-564
https://wikia-inc.atlassian.net/browse/CONCF-570
https://wikia-inc.atlassian.net/browse/CONCF-571
https://wikia-inc.atlassian.net/browse/CONCF-572

### CODE REVIEW

**Comment by hakubo** [comment]
LGTM but
we'll need time to refactor most of this soon

**Comment by vforge** [comment]
Yes, we need.

File: extensions/wikia/AdminDashboard/css/AdminDashboard.scss [hunk 1]
```diff
@@ -528,10 +529,12 @@ $color-highlight-column: rgba($color-page, 0);
...
 
 	.mw-special-AdminDashboard .WikiaMainContentContainer {
-		margin-right: $admin-dashboard-rail-width;
+		@if ( $responsive-layout-enabled ) {
+			margin-right: $admin-dashboard-rail-width;
+		}
 		padding: 0;
 
-		@media screen and (max-width: $breakpoint-small) {
+		@media screen and (max-width: 1023px) {
```
Comment (Warkot) [inline extensions/wikia/AdminDashboard/css/AdminDashboard.scss]: Why did you remove this variable? I liked it. </3

Comment (vforge) [inline extensions/wikia/AdminDashboard/css/AdminDashboard.scss]: It's same value in responsive and breakpoints, but `$breakpoint-small` variable is unavailable in breakpoints - instead of importing two different layouts I've done this quick fix. All "fixes" will be reviewed in later ticket.

**Comment by Warkot** [comment]
~~LGTM~~ looks mergable to me :water_buffalo:

File: extensions/wikia/AdminDashboard/css/AdminDashboard.scss (hunk 1 above)
Comment (Warkot) [inline extensions/wikia/AdminDashboard/css/AdminDashboard.scss]: Looking forward into it. :eyeglasses:

**Comment by hakubo** [comment]
LGTM
